tests/agent/test_prompt.py

```
@@ -0,0 +1,250 @@
+
+    if config["api_key"] is None:
+        pytest.skip(f"API key not available for {config['provider']}")
+
+    full_prompt = ELEMENTWISE_ADD_PROMPT + "\n\n" + FFI_PROMPT_SIMPLE
+
+    if config["provider"] == "openai":
+        response = call_openai_model(config["model"], config["api_key"], full_prompt)
+    elif config["provider"] == "anthropic":
+        response = call_anthropic_model(config["model"], config["api_key"], full_prompt)
+    else:
+        raise ValueError(f"Unknown provider: {config['provider']}")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)